content/docs/rosa/prereq-list/index.md

```diff
@@ -0,0 +1,103 @@
+
+- Ensure that your organization's SCP are not more restrictive than the roles and policies required by the cluster.
+
+- Ensure that your SCP is configured to allow the required `aws-marketplace:Subscribe` permission when you choose `Enable ROSA` from the console, and please refer [here](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshoot-rosa-enablement.html#error-aws-orgs-scp-denies-permissions) for more details.
+
+- When you create a ROSA cluster using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is created as well. 
+    - This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. 
+    - Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.
+
+
+## Networking Prerequisites
+Next, let's talk about the prerequisites needed from networking standpoint.
+
+### Firewall
+  - Configure your firewall to allow access to the domains and ports listed [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs)
+### Custom DNS
+  - If you want to use custom DNS, then ROSA installer must be able to use VPC DNS with default DHCP options so it can resolve hosts locally. 
+      - To do so, run `aws ec2 describe-dhcp-options` and see if the VPC is using VPC Resolver.
+      - Otherwise, the upstream DNS will need to forward the cluster scope to this VPC so the cluster can resolve internal IPs/services.
+
+## PrivateLink Prerequisites
+If you would like to deploy a PrivateLink cluster, then be sure to deploy the cluster in the pre-existing VPC (BYO VPC) and please refer [here](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-vpc.html) for more details and below in high level:
+
+- Create a public and private subnet for each AZ that your cluster uses.
+    - Alternatively, implement transit gateway for internet/egress with appropriate routes.
+- The VPC's CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address for cluster machines. 
+    - The subnet CIDR blocks must belong to the machine CIDR that you specify.
+- The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
+- Set both `enableDnsHostnames` and `enableDnsSupport` to `true`.
+    - That way, the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster’s internal DNS records.
+- Verify route tables by running `aws ec2 describe-route-tables --filters "Name=vpc-id,Values=<vpc-id>"`. 
+    - Ensure that the cluster can egress either via NAT gateway in public subnet or via transit gateway.
+    - And ensure whatever UDR you would like to follow is set up.
+    
+
+## ROSA Prerequisites without STS 
```

Comment:
We should tell them that nonSTS clusters are deprecated and soon new clusters will only be STS.  Don't encourage them to go do mint mode.

Reply:
i'll just chop this part off